manifests/telegraf/agent.pp

```diff
@@ -312,7 +316,7 @@
     }
 
     # The port to use for this mbean is 8143 for Orchestrator and 8140 for Puppet server
-    if puppet_operational_dashboards::include_pe_metrics {
```

Comment:
two bugs:
* `$` was missing
* the class doesn't depend on the init.pp so it's not aware of the `$puppet_operational_dashboards::include_pe_metrics` variable.